rand_core/src/lib.rs

```diff
@@ -35,7 +35,7 @@
 
 #![doc(html_logo_url = "https://www.rust-lang.org/logos/rust-logo-128x128-blk.png",
        html_favicon_url = "https://www.rust-lang.org/favicon.ico",
-       html_root_url = "https://docs.rs/rand_core/0.1.0")]
+       html_root_url = "https://docs.rs/rand_core/0.2")]
```

Comment:
This was mentioned in the top-level Cargo.toml but not the rand_core one...

Reply:
[Are you sure...?](https://github.com/rust-lang-nursery/rand/blob/master/rand_core/Cargo.toml#L3)